flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
this is disappointing. Why denying non "intel" users?

Reply:
Actually I don't know for sure if OpenSeeFace supports aarch64 as this is a wapper UI for it and I thought providing a UI which works on ARM but not the actual thing it launches might be a bit misleading.

At this point I have not tested this project on ARM.

Reply:
it's you to answer that question

Also https://github.com/flathub/flathub/pull/5394/files#r1667700973